srcpkgs/btop/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'btop'
+pkgname=btop
+version=1.0.24
```

Comment:
1.1.3 is out

Reply:
I'd been procrastinating updating this because 1.1.0 broke cross and I haven't had time to work out how to fix it, I've pushed an updated template so it's clear that it's (hopefully temporarily) broken.